src/Omega_h_meshsim.cpp

```diff
@@ -621,17 +647,30 @@ Mesh read(filesystem::path const& mesh_fname, filesystem::path const& mdl_fname,
   pProgress p = NULL;
   pGModel g = GM_load(mdl_fname.c_str(), nm, p);
   pMesh m = M_load(mesh_fname.c_str(), g, p);
+  const bool hasNumbering = (numbering_fname.native() != std::string(""));
+  pMeshNex numbering = hasNumbering ? MeshNex_load(numbering_fname.c_str(), m) : NULL;
```

Comment:
nullptr

Reply:
Thank you.  This is fixed here https://github.com/SCOREC/omega_h/pull/40/commits/3aed91a71a5122a2ebd8c84b1f9c4f0bc8d53433.